src/Symfony/Bridge/Twig/ErrorRenderer/TwigErrorRenderer.php

```diff
@@ -41,17 +41,17 @@ public function __construct(Environment $twig, HtmlErrorRenderer $fallbackErrorR
 
     public function render(\Throwable $exception): FlattenException
     {
-        $exception = $this->fallbackErrorRenderer->render($exception);
-        $debug = \is_bool($this->debug) ? $this->debug : ($this->debug)($exception);
+        $flattenException = $this->fallbackErrorRenderer->createFlattenException($exception);
```

Comment:
Maybe we don't have to create a dedicated method for it. Instead, we can just call to `FlattenException::createFromThrowable()` directly. Anyway, what we need here is the status code info only.

Full suggestion:
```php
public function render(\Throwable $exception): FlattenException
{
    $flattenException = FlattenException::createFromThrowable($exception);
    $debug = \is_bool($this->debug) ? $this->debug : ($this->debug)($flattenException);

    if ($debug || !$template = $this->findTemplate($flattenException->getStatusCode())) {
        return $this->fallbackErrorRenderer->render($exception);
    }

    return $flattenException->setAsString($this->twig->render($template, [
        'exception' => $flattenException,
        'status_code' => $flattenException->getStatusCode(),
        'status_text' => $flattenException->getStatusText(),
    ]));
}
```
For the 2nd scenario the `Content-Type` header will be set on `Response::prepare()` afterward. Wdyt?

Reply:
Thanks for the review, I've applied your patch 👍🏼